.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Since this is the first GHA workflow, can you please also add the dependabot config (https://github.com/freedomofpress/securedrop/pull/7102)

Reply:
Done!